File: s02e01_intro/carro.py
class Carro:
    nPessoas = 0
    maxPessoas = 2
    gasolina = 0
    def embarcar(self):
        if self.nPessoas < self.maxPessoas:
            self.nPessoas += 1
            print("embarcou")
            return True
        print("ta lotado")
        return False

carro = Carro()
carro2 = carro #
carro.embarcar()
carro.embarcar()
carro.embarcar()
print(carro.nPessoas)
print(carro2.nPessoas)

File: s02e01_intro/Carro.java
package s02e01_intro;

public class Carro{
    int nPessoas = 0;
    int maxPessoas = 2;
    float gasolina = 0;

    boolean embarcar(){
        if (this.nPessoas < this.maxPessoas){
            this.nPessoas += 1;
            System.out.println("embarcou");
            return true;
        }
        System.out.println("ta lotado");
        return false;
    }

    public static void main(String[] args) {
        Carro carro = new Carro();
        Carro carro2 = carro;
        carro.embarcar();
        carro.embarcar();
        carro.embarcar();
        System.out.println(carro.nPessoas);
        System.out.println(carro2.nPessoas);
    }

}


